modules/renter/contractor/editor.go

```diff
@@ -0,0 +1,209 @@
+package contractor
+
+import (
+	"bytes"
+	"errors"
+	"net"
+	"time"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+	"github.com/NebulousLabs/Sia/modules"
+	"github.com/NebulousLabs/Sia/types"
+)
+
+const (
+	// SectorSize is the number of bytes in a sector.
+	SectorSize = 1 << 22 // 4 MiB
+)
```

Comment:
needs to be moved to the modules package. Though, it'll screw up my code if you do immediately, so instead you should drop a TODO here. I believe that my settings are 

4kb for testing
1MiB for dev
4Mib for release